app/config/permissions.neon

```diff
@@ -105,10 +105,17 @@ permissions:
           - update
           - remove
           - addSubgroup
-          - addSupervisor
           - removeSupervisor
       conditions: group.isAdmin
 
+    - allow: true
```

Comment:
This lets the superadmin make a student supervisor of a group. I'm not sure if that's okay...

Reply:
Well, thats true, but according to this we should do the checks on supervisor role in set-role endpoint itself?